purplesmurf1998/Autotracks
# Sprint 4 inventory view

# Description
This pull request consists of mainly one user story, the ability to view the inventory with the proper columns depending on the vehicle properties set in the dealership

- [AP-12] As a user, I would like to see the full list of vehicles in the dealership

A staff member with the proper authorization will be able to see the inventory table for their dealership.
An admin can switch between their dealerships to view their respective inventories separately.
An admin can set a default dealership, so that they don't have to select one every time.
Only the vehicle properties set as visible will appear as columns in the table, and in the order they are saved as.
# How to test

1. Log in as an administrator or staff user
2. Navigate to the inventory page view
3. If staff member, you should see the proper columns appear according to your dealership
4. If admin, you can select between your various dealerships
5. Click the top right button "Set selected dealership as default" to set the currently selected dealership as the default one. (This button only appears if you are admin, and if the currently selected dealership is not the current default one)
6. As admin, you can modify vehicle properties in the dealership details page and see the reflected changes in the inventory table.
# Screenshots
<img width="1440" alt="Screen Shot 2021-11-26 at 12 41 19 PM" src="https://user-images.githubusercontent.com/44242944/143616328-5785e269-a9c0-4794-ab65-9b525cb0d6bc.png">
<img width="1440" alt="Screen Shot 2021-11-26 at 12 41 36 PM" src="https://user-images.githubusercontent.com/44242944/143616352-e2dfbb54-e4f6-47fa-973d-58f1fe49833d.png">
<img width="1440" alt="Screen Shot 2021-11-26 at 12 41 52 PM" src="https://user-images.githubusercontent.com/44242944/143616360-50480588-2ea1-471b-b80b-95f4fce8d483.png">
